tests/test_proc/test_basic.py

```diff
@@ -66,3 +67,12 @@ def test_max(self, random_patch):
         axis = dims.index("time")
         norm = np.abs(np.sum(time_norm.data, axis=axis))
         assert np.allclose(norm, 1)
+
+
+class TestStandarize:
+    """Tests for standardization."""
+
+    def test_base_case(self, random_patch):
+        """Ensure runs with default parameters."""
+        out = random_patch.standardize("time")
+        assert not np.any(pd.isnull(out.data))
```

Comment:
Can you also add an assert that all the standard deviations along the time dimension are unity?

Reply:
Done!